src/tools/run_onnx_model/run_onnx_model.cpp

```diff
@@ -55,8 +56,9 @@ int load_inputs(vector<string> input_paths, vector<shared_ptr<runtime::Tensor>>
     for (int i = 0; i < input_paths.size(); i++)
     {
         const string input_path = input_paths.at(i);
-        std::vector<float> data = read_binary_file<float>(input_path);
-        copy_data(inputs.at(i), data);
+        std::vector<float> data = ngraph::file_util::read_binary_file<float>(input_path);
```

Comment:
This line will cause problems on quantized models but since this is a rare case it isn't a blocker for this PR.